CHANGELOG.md

```diff
@@ -2,15 +2,14 @@
 
 ## Unreleased
 
-### Fixes
-
 ### Changes
 
 * Bolsters the CSS that backs Apostrophe UI's typography to help prevent unintended style leaks at project-level code.
 
 ### Fixes
```

Comment:
We've been putting Fixes first, I think to highlight those. Maybe we don't need to do that?